src/contributing.md

```diff
@@ -1,17 +1,10 @@
 # Contributing to Rust
 
+<!-- toc -->
+
 Thank you for your interest in contributing to Rust! There are many ways to
 contribute, and we appreciate all of them.
```

Comment:
Hmm, I kind of like that the table of contents came after 'there are many ways to contribute', it's like it's showing the ways to contribute. If you move `<!-- toc -->` after the paragraph, will it render the table of contents after too?

Reply:
Yep! it renders the TOC wherever the marker is, I'll push a change moving it back before merging.